tests/lowering/misc/test_select.py

```diff
@@ -0,0 +1,60 @@
+import torch
+import torch_ttnn
+import pytest
+import ttnn
+
+from tests.utils import assert_with_pcc
+
+
+class SelectModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, input, dim):
+        return torch.select(input, dim, 0)
```

Comment:
Maybe also test with different index beyond 0?

Reply:
Sure, refer to 87a3a609ae5e513f380e23eb06a92d943f569c62 adding one more `index` parameter. And more test cases for different index.